### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: common/Makefile.common.mk [hunk 1]
```diff
@@ -96,15 +96,15 @@ update-common:
...
+	@cp -a $(TMP)/common-files/files/* $(shell pwd)
 	@rm -fr $(TMP)/common-files
 
 update-common-protos:
 	@mkdir -p $(TMP)
 	@git clone -q --depth 1 --single-branch --branch $(UPDATE_BRANCH) https://github.com/istio/common-files $(TMP)/common-files
 	@cd $(TMP)/common-files ; git rev-parse HEAD > common-protos/.commonfiles.sha
 	@rm -fr common-protos
-	@cp -ar $(TMP)/common-files/common-protos $(shell pwd)
+	@cp -a $(TMP)/common-files/common-protos $(shell pwd)
```
Comment (nmittler) [inline common/Makefile.common.mk]: Why remove `-r`?

Comment (jwendell) [reply common/Makefile.common.mk]: this comes from common-files repo. -a includes -r already.

Comment (nmittler) [reply common/Makefile.common.mk]: oh right, of course

**Pull request merged**